elastio-s3-changelog/README.md

```diff
@@ -11,11 +11,18 @@ Then, the Elastio `iscan` job reads those events to perform the scanning of new
 
 1. First, you need to enable Amazon EventBridge for your S3 buckets by following these instructions:
     [Enabling Amazon EventBridge](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-event-notifications-eventbridge.html).
-2. You can use
-    [this quick-create link](https://us-east-2.console.aws.amazon.com/cloudformation/home?region=us-east-2#/stacks/create/review?templateURL=https://elastio-prod-artifacts-us-east-2.s3.us-east-2.amazonaws.com/contrib/elastio-s3-changelog/v1/cloudformation-multiple-buckets.yaml&stackName=elastio-s3-changelog)
-    to create the stack.
-
-    > You might need to switch to the region where your Elastio CloudFormation stack is deployed.
+2. Use one of the following quick-create links. Choose the region where your Elastio CloudFormation stack is deployed:
```

Comment:
Let's sort the regions in order of their popularity. I believe it should be smth like this:
- US regions
- EU regions
- CA regions
- AP regions

Also the wording is wrong. 

```suggestion
2. Use one of the following quick-create links. Choose the region where your Elastio connector is deployed:
```

Reply:
Technically it's called "Elastio Cloud Connector" )